requirements/project-requirements.txt

```diff
@@ -91,8 +91,10 @@ wad==0.4.6
 debloat==1.6.4
 phonenumbers==9.0.3
 die-python==0.4.0
-guarddog==2.1.0 # version greater than 2.1.0 raises dependency conflicts
+# guarddog==2.1.0 # version greater than 2.1.0 raises dependency conflicts. Commenting this out due to dependency conflicts.
```

Comment:
I am fine with this but I think we should disable this Guarddog analyzer by default until there is a valid solution to avoid users running into it and getting fails.

Reply:
yep!